packages/modules/src/modules/header/EoYUSMomentHeader.stories.tsx

```diff
@@ -5,6 +5,12 @@ import { Header } from './EoYUSMomentHeader';
 import { css } from '@emotion/core';
 import { brand } from '@guardian/src-foundations';
 
+const givingTuesdayStart = new Date('2021-11-29T17:00:00'); //remove "Subscribe" Monday 12:00 PM EST
+const givingTuesdayEnd = new Date('2021-11-01T09:00:00'); //re-add "Subscribe" on Wednesday morning GMT
+const currentDateTime = new Date();
+const shouldShowSubscribeButton =
+    givingTuesdayStart <= currentDateTime && currentDateTime <= givingTuesdayEnd;
+
```

Comment:
Nothing wrong with the logic here! I'd say we probably don't need to do this in the `.stories` file though. If we want to be able to see the component in both forms here I'd recommend creating a new story in this file e.g

```ts
export const DefaultGivingTuesdayHeader = Template.bind({});
// ...set the props to not have a secondary cta...
```

I don't think we need to make that change in this PR, perhaps just useful going forwards though!

Reply:
Thanks Tom, we thought as it was a temp change we'd pop it in the existing stories file but have spoken about making a separate file!

Reply:
I'd recommend leaving it in this file - I'd just add a new story at the bottom!

Reply:
Ah yes sorry storybook component!